packages/gatsby-theme-carbon/src/components/EditLink/EditLink.js

```diff
@@ -1,40 +1,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { css } from '@emotion/core';
+import { graphql, useStaticQuery } from 'gatsby';
 
-import { link } from './EditLink.module.scss';
+import { link, row } from './EditLink.module.scss';
 
-const repository = 'https://github.com/temporary/master/tree';
+const EditLink = ({ relativePagePath, repository: repositoryProp }) => {
```

Comment:
Moved this to it's own query. Ideally data fetching would be separate from the component itself but we definitely don't want them sharing queries in case one gets shadowed.